arcgis-ios-sdk-samples/Layers/Colormap renderer/ColormapRendererViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ColormapRendererViewController: UIViewController {
+    @IBOutlet private weak var mapView: AGSMapView! {
```

Comment:
Some samples use access control to make properties and methods `private`, but it makes no practical difference. The default of `internal` is just fine.